frontend/test/metabase/scenarios/embedding/embedding-full-app.cy.spec.js

```diff
@@ -97,6 +114,181 @@ describe("scenarios > embedding > full app", () => {
       cy.button("Summarize").should("not.exist");
       cy.button("Filter").should("not.exist");
     });
+
+    describe("desktop logo", () => {
+      it("should be able to toggle side nav", () => {
+        // default logo: true
+        visitQuestionUrl({ url: "/question/1", qs: { side_nav: true } });
+        cy.findByRole("banner").within(() => {
+          cy.findByText(/Our analytics/i).should("be.visible");
+          cy.findByTestId("main-logo").should("be.visible");
+        });
+
+        cy.findByTestId("sidebar-toggle").click();
+        cy.findByText(/Add your own data/i).should("be.visible");
+
+        cy.findByRole("banner").within(() => {
+          cy.findByText(/Our analytics/i).should("not.be.visible");
+          cy.findByTestId("main-logo").should("be.visible");
+        });
+      });
+
+      it("should hide side nav toggle icon on hover", () => {
+        // default logo: true
+        visitQuestionUrl({ url: "/question/1", qs: { side_nav: false } });
+        cy.findByRole("banner").within(() => {
+          cy.findByText(/Our analytics/i).should("be.visible");
+          cy.findByTestId("main-logo").should("be.visible");
+        });
+
+        cy.findByTestId("sidebar-toggle").should("not.exist");
+      });
+
+      it("should always show side nav toggle icon when logo is hidden", () => {
+        visitQuestionUrl({
+          url: "/question/1",
+          qs: { side_nav: true, logo: false },
+        });
+        cy.findByRole("banner").within(() => {
+          cy.findByText(/Our analytics/i).should("be.visible");
+          cy.findByTestId("main-logo").should("not.be.visible");
+          cy.icon("sidebar_closed").should("be.visible");
+        });
+        // Non-hovered color
+        cy.findByTestId("sidebar-toggle")
+          .children()
+          .should("have.css", "color", toRgbString(color("text-medium")));
```

Comment:
when checking the style, I'm not totally sure why it is rendered with rgb string. 🤷

Reply:
Is generally expected to not test style in E2E? This could be a requirement, at last from UX's point of view. https://metaboat.slack.com/archives/C045W5MF84U/p1671814108759669?thread_ts=1671734932.320549&cid=C045W5MF84U

I'm fine removing this. But do we need to even substitute it with a unit test or removing it is just fine?